src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js

```diff
@@ -444,6 +465,8 @@ function redirect3ds2() {
  * @returns rendering template or error
  */
 function authorize3ds2() {
+  Logger.getLogger('Adyen').error('entering authorize3ds2 3ds2');
```

Comment:
same here, an orderNo would be useful to log.

Reply:
Actually this one is not needed